scout/adapter/mongo/variant.py

```diff
@@ -204,6 +204,19 @@ def variants(
 
         return result
 
+    def variants_count(self, case_id, query=None, variant_ids=None, category="snv"):
+        """Count number of variants
```

Comment:
Could you elaborate this docstring a bit more? You could follow the general convention that we use in Scout. It contains a short description, then Args and  Returns

Reply:
Function deleted.